envdef.go

```diff
@@ -0,0 +1,22 @@
+package cwl
+
+// RequirementEnvDef only appears if requirement class is "InitialWorkDirRequirement"
+type EnvDef struct {
+        envName  string
+        envValue string
```

Comment:
1. Is it really necessary to be private fields? It could be public, IMO
2. `env` prefix is not needed. Just `Name` and `Value` might make sense.